infrastructure/web/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.50.0"
+    }
+  }
```

Comment:
When we start using this properly we'd want to store the state file in the Terraform Cloud, rather than locally.

Inside the terraform block we want a [cloud block](https://developer.hashicorp.com/terraform/cli/cloud/settings):

```terraform
  cloud {
    organization = "oak-national-academy"
    workspaces {
      name = "app-name-${var.environment}"
    }
```
Obvs changing `app-name` to something relevant to the app this is to be part of.

Reply:
Note. I don't think you have access to Terraform Cloud atm, but I am trying to negotiate getting more people access to Terraform Cloud so hopefully that will change.